shop/src/pages/admin/products/Edit.js

```diff
@@ -329,7 +328,6 @@ const EditProduct = () => {
 
       // Clear memoize cache for existing product
       fetchProduct.cache.delete(`${config.dataSrc}-${product.id}`)
-      fetchProductStock.cache.clear()
```

Comment:
Wasn't there a good reason for clearing the in-memory cache?
What is the impact of not clearing the cache anymore>

Reply:
> Wasn't there a good reason for clearing the in-memory cache?

`fetchProductStock` was a memoized function until it was determined that it being just an async function worked better for DShop: https://github.com/OriginProtocol/dshop/pull/956/commits/60255ad22383aaf2bda406565f84bc7481e3b3f7
My understanding is that a regular async function will not have a _cache_ property to clear.

Reply:
ah yes, thanks for digging the details.